docs/src/threading.rst

```diff
@@ -78,6 +78,14 @@ Threads
 
     .. versionchanged:: 1.4.1 returns a UV_E* error code on failure
 
+.. c:function:: int uv_thread_detach(uv_thread_t* tid)
+
+    Detaches a thread. Detached threads automatically release their
+    resources upon termination, eliminating the need for the application to
+    call `uv_thread_join`.
+
+    .. versionadded:: REPLACEME
```

Comment:
I don't believe libuv has any support for `REPLACEME` (https://github.com/libuv/libuv-release-tool/issues/7).

Reply:
Should I use what is in `version.h`?

Reply:
Based on https://github.com/juanarbol/libuv/commit/0eca049a9b3ca7440b4a4c00ab0ebe57cc3ff948 I should be using next patch version

Reply:
#4599 is currently using 1.50.0. Since this seems likely to land in time for the next version, I think it makes sense to use that here.